reference/platform-app-yaml.md

```diff
@@ -390,8 +398,10 @@ relationships:
     solr: "solr:solr"
     redis: "redis:redis"
 web:
-    document_root: "/"
-    passthru: "/index.php"
+    locations:
+        "/":
+            root: "public"
+            passthru: "/index.php"
```

Comment:
The confusion around that is exactly what I'm trying to sort out and properly document at the moment. :smile: